react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,20 @@
+import logo from './logo.svg';
+import './App.css';
+import './CustomButton.css';
+
+function App() {
+  return (
+    <div className="App">
+      <CustomButton text={'I'} color={'red'} />
+      <CustomButton text={'know'} color={'green'} />
+      <CustomButton text={'React!'} color={'blue'} />
+    </div>
+  );
+}
+
+function CustomButton({text, color}) {
+
+  return <button className = {color}>{text}</button>
+}
```

Comment:
This should be inside it's own file. Remember that we don't create multiple components in the same file. Each component gets it's own file